src/oc/web/components/comments.cljs

```diff
@@ -14,8 +18,26 @@
             [goog.events :as events]
             [goog.events.EventType :as EventType]))
 
-(rum/defc comment-row < rum/static
-  [c]
+(defn delete-clicked [e comment-data]
+  (let [alert-data {:icon "/img/ML/trash.svg"
+                    :action "delete-comment"
+                    :message (str "Delete this comment?")
+                    :link-button-title "No"
+                    :link-button-cb #(dis/dispatch! [:alert-modal-hide])
+                    :solid-button-title "Yes"
+                    :solid-button-cb #(let [org-slug (router/current-org-slug)
+                                            board-slug (router/current-board-slug)
+                                            board-url (utils/get-board-url org-slug board-slug)
+                                            activity-uuid (router/current-activity-id)]
+                                       (router/nav! (oc-urls/entry org-slug board-slug activity-uuid))
```

Comment:
@thepug why do you navigate to the entry again? you are already here, maybe to force comments reload?